### PR TITLE
Update platform band

### PR DESCRIPTION
**Pull request by oraNod**
Fixes issue #113 and makes some other improvements for the platform band that make it scale better.

### CODE REVIEW

**Review by GregSutcliffe — DISMISSED** [review]
Renders fine here, feel free to merge once you resolve the conflict

**Comment by oraNod** [comment]
> Renders fine here, feel free to merge once you resolve the conflict

@GregSutcliffe Unfortunately fixing the conflict meant dismissing your previous approval. I don't want to merge without at least one recorded approval. Thanks.

**Comment by GregSutcliffe** [comment]
Commendable dedication to process! Merged, thanks :)

**Pull request merged**
